isis/src/base/apps/spiceinit/spiceinit.cpp

```diff
@@ -662,12 +657,17 @@ namespace Isis {
         continue;
       }
     }
+    
+    if (QString(ui.GetString("SHAPE")).toLower() == "user") {
+      kernelsGroup["ShapeModel"] = ui.GetAsString("MODEL");
```

Comment:
Not sure why GetAsString is being used here. Why not use getFile? The MODEL parameters is listed as type FILE. NOTE: Get as string is normally only used by the GUI to retrieve all values regardless of type.